pkg/capi/aws.go

```diff
@@ -143,6 +143,81 @@ var _ = Describe("Cluster API AWS MachineSet", framework.LabelCAPI, framework.La
 		Expect(err).ToNot(HaveOccurred(), "Failed to create CAPI machineset")
 		framework.WaitForCAPIMachinesRunning(ctx, cl, machineSet.Name)
 	})
+
+	//OCP-78677 - [CAPI] Dedicated tenancy should be exposed on aws providerspec.
+	It("should be able to run a machine with dedicated instance", func() {
+		awsMachineTemplate = newAWSMachineTemplate(mapiDefaultProviderSpec)
+		awsMachineTemplate.Spec.Template.Spec.Tenancy = "dedicated"
+		Expect(cl.Create(ctx, awsMachineTemplate)).To(Succeed(), "Failed to create awsmachinetemplate")
+		machineSetParams = framework.UpdateCAPIMachineSetName("aws-machineset-78677", machineSetParams)
+		machineSet, err = framework.CreateCAPIMachineSet(ctx, cl, machineSetParams)
+		Expect(err).ToNot(HaveOccurred(), "Failed to create CAPI machineset")
+		framework.WaitForCAPIMachinesRunning(ctx, cl, machineSet.Name)
+	})
+
+	//huliu-OCP-75662 - [CAPI] AWS Machine API Support of more than one block device.
+	It("should be able to run a machine with more than one block device", func() {
+		awsMachineTemplate = newAWSMachineTemplate(mapiDefaultProviderSpec)
+		encryptBool := true
+		unencryptBool := false
+		awsMachineTemplate.Spec.Template.Spec.NonRootVolumes = []awsv1.Volume{
+			{
+				DeviceName: "/dev/xvda",
+				Size:       120,
+				Type:       awsv1.VolumeTypeGP3,
+				IOPS:       5000,
+				Encrypted:  &encryptBool,
```

Comment:
You could use the `ptr.To` pattern here to inline the booleans, minor nit

Ref https://pkg.go.dev/k8s.io/utils/ptr